Gemfile

```diff
@@ -37,6 +38,7 @@ group :development, :test do
   gem 'dotenv-rails'
   gem 'rspec-rails'
   gem 'guard-rspec'
+  gem 'active_record_doctor'
```

Comment:
What's this used for?

Reply:
It's used to identify potential issues within an Active Record DB. I was using it when testing performance of the import, but I can remove it.

Reply:
commit https://github.com/openbrewerydb/openbrewerydb-rails-api/pull/68/commits/9b5e26a749222b22247b172bd132448cd7aac6bf